draft-ietf-quic-tls.md

```diff
@@ -613,16 +613,27 @@ by the "initial_max_data" transport parameter supplied by the server.  A client
 MUST treat receipt of a NewSessionTicket that contains an "early_data" extension
 with any other value as a connection error of type PROTOCOL_VIOLATION.
 
+A client that wishes to send early data uses the "early_data" extension in the
```

Comment:
Maybe s/early data/0-RTT packets/? I do not think QUIC sends early data.

Reply:
Done.

Reply:
```suggestion
A client that wishes to send 0-RTT packets uses the "early_data" extension in the
```